tests/output/unix/filter-when-using-tags-with-args.txt

```diff
@@ -0,0 +1,24 @@
+[36m@foo(bar)
```

Comment:
i think we also need a file for windows similar to this? did you run tests on windows?

Reply:
Yeah haha I was hoping to get the windows output from a failing CI instead of switching computers. I'll get that tonight as well

Reply:
fair no idea why they don't run on your MR something else i need to checkout :sweat_smile: but feel free to not do the windows then i will take care of it :)

Reply:
you should be able to rebase to get a pipeline run